brave-spancollector-scribe/src/test/java/com/github/kristofa/brave/scribe/ScribeReceiver.java

```diff
@@ -81,4 +77,11 @@ public ResultCode Log(final List<LogEntry> messages) throws TException {
         return result;
     }
 
+    public void setDelayMs(int delayMs) {
```

Comment:
The field is private, originally it was provided in the constructor (it was dead code though, nobody used this feature). I'd prefer to set it via setter where needed to have a common scribe server setup in my newly added test class.